src/main/java/uk/gov/hmcts/reform/cpo/validators/IdsValidator.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.reform.cpo.validators;
+
+import uk.gov.hmcts.reform.cpo.validators.annotation.ValidCpoId;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+public class IdsValidator implements ConstraintValidator<ValidCpoId, Optional<List<String>>>, Validator <String>{
```

Comment:
Dan if I put List<UUID> and lets Spring to take care . It will response 400 error without messages and log info when the endpoint is called with  ids=b00445ee-5bed-42c5-812f-12687175becaXXXX&c